docs/source/troubleshooting/self_verification.rst

```diff
@@ -1,7 +1,7 @@
 Running Self-Verification
 =========================
 
-|st2| includes a script to verify the installation, using |st2| itself.
+Brocade Workflow Composer includes a script for verifying the BWC installation.
```

Comment:
The docs build process gets a bit complicated, but there's a few things going on here.

One is that we use the same base documentation for docs.stackstorm.com and bwc-docs.brocade.com. The difference is that the bwc-docs have a 'Brocade' theme applied, and the Automation Suites documentation gets merged in.

Another key difference is that `|st2|` gets automatically replaced by "StackStorm" on docs.stackstorm.com, and by "BWC" on bwc-docs.brocade.com. `|fullname|` gets replaced by "StackStorm" on docs.stackstorm.com, and "Brocade Workflow Composer" on bwc-docs.

In general, we want to refer to the product as "StackStorm" on docs.stackstorm.com, and "BWC/Brocade Workflow Composer" on bwc-docs.brocade.com. So in this specific case, you should use either `|st2|` or `|fullname|`.

Reply:
Thank you. Update made per comment. See new commit changes.